lib/kcerts/ssh.go

```diff
@@ -96,9 +97,27 @@ func (a SSHAgent) Valid() bool {
 	return err == nil
```

Comment:
just notice the line above, probably need to add an `if err != nil` before `defer` and `close`? I suspect the code as is will work, but not sure the fact that close works on a nil conn is specified behavior.

Reply:
It does work as intended because the defer would swallow the nil exception, but it should catch the error, at least to be idiotmatic. Added in a err handle.